include/openenclave/bits/evidence.h

```diff
@@ -25,6 +24,12 @@ OE_EXTERNC_BEGIN
     }
 #define X509_OID_FOR_OE_EVIDENCE_STRING "1.2.840.113556.10.1.2"
 
+#define X509_OID_FOR_NEW_OE_EVIDENCE_EXT                     \
```

Comment:
I didn't see this define is used anywhere in the code. Am I missing something?

Reply:
X509_OID_FOR_NEW_OE_EVIDENCE_EXT is a recently agreed upon OID for evidence extension. It will be used for certificate generation in the next release. We want to ensure the verifiers can handle such OID before starting generating with the new OID.